pwndbg/gdblib/events.py

```diff
@@ -179,15 +179,19 @@ def log_objfiles(ofile=None) -> None:
 
 def after_reload(start=True) -> None:
     if gdb.selected_inferior().pid:
-        for f in registered[gdb.events.stop]:
-            f()
-        for f in registered[gdb.events.start]:
-            if start:
+        if gdb.events.stop in registered:
+            for f in registered[gdb.events.stop]:
```

Comment:
can we do `for f in registered.get(gdb.events.stop, []):`?

Reply:
I'm up with either solution, pls send another PR if u think this suggestion makes sense :)